examples/kubernetes/dynamic-scheduling/README.md

```diff
@@ -0,0 +1,29 @@
+# Dynamic Volume Provisioning with AWS EBS CSI Driver
```

Comment:
NIT:

```suggestion
# Dynamic Volume Provisioning
```

Since the example is under AWS EBS CSI Driver, we could skip mentioning this